vnext/src/Libraries/Components/Pressable/Pressable.windows.js

```diff
@@ -34,6 +34,7 @@ import type {
 import type {HandledKeyboardEvent} from '../../Components/View/ViewPropTypes';
 import View from '../View/View';
 import TextInputState from '../TextInput/TextInputState';
+import {number} from 'prop-types';
```

Comment:
This should be removed. `number` is a built-in primitive in flow.

The import shadows that, with a different `number` type, with a different usage. See https://www.npmjs.com/package/prop-types

Reply:
good to know